tests/cross_validation/c_harness/harness.c

```
@@ -0,0 +1,185 @@
+  sd->data[0] = F1; sd->data[1] = F2;
+  p->pulsar.spindown = sd;
+  p->site = &lalCachedDetectors[LHODIFF];
+  p->ephemerides = edat;
+  p->startTimeGPS.gpsSeconds = START_GPS;
+  p->startTimeGPS.gpsNanoSeconds = 0;
+}
+
+static void dump(const char *path, const REAL4TimeSeries *ts) {
+  FILE *f = fopen(path, "wb");
+  UINT4 n = ts->data->length;
+  fwrite(&n, sizeof(UINT4), 1, f);
```

Cppcheck (2.21.0)

[warning] If resource allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfResources)